### ISSUE TITLE
Any plans for publishing checkpoints/code of the LTRe model ?

### ISSUE THREAD

**Issue by jobergum** [jingtaozhan/RepBERT-Index#3]
Thanks a lot for sharing this work and I have successfully reproduced the MRR@10 metrics on the MS Marco Passage ranking task using your published checkpoint.  I'm wondering if you are planning on sharing the checkpoint or code on how to reproduce the new LTRe model which achieves better MRR accuracy after training.  https://arxiv.org/abs/2010.10469

Again, thanks a lot for sharing this work.

**Comment by jingtaozhan** [comment]
Thank you for your attention to this work.

I'm very happy that you also notice our recent work, which we believe is a very good combination of performance and latency.

Personally speaking, since LTRe is neither officially published nor on the MS Marco leaderboard, I think not many people notice this work. Thus, I haven't worked on my 'messy' code yet 🤦‍♂️. I will share it when it is officially published or it catches many researchers' attention. 

BTW, recently I trained another document encoder with a very simple method and use LTRe to further train the query encoder. It achieves much better performance than the LTRe(ANCE) model reported in the arxiv paper. 

May I ask how you noticed LTRe?

**Comment by jobergum** [comment]
Thank you for the quick reply, 

I found this repo and the original paper from search 👍  The results in the LTRe paper is pretty close to SOTA for dense single state retrieval on MS Marco passage. Looking forward to your publication and MS Marco submission. I work on Vespa.ai and we have worked with dense retrievers for question answering apps (e.g see https://blog.vespa.ai/efficient-open-domain-question-answering-on-vespa/) but currently looking at doing a sample app for MS Marco passage/document ranking.
